assets/js/componentsEnvironmentDetails.js

```diff
@@ -23,13 +24,31 @@ jQuery(function () {
           build_image_tag: env.build_image_tag,
           alert_severity_label: env.alert_severity_label === null ? 'Not set' : env.alert_severity_label,
           alerts_slack_channel: env.alerts_slack_channel === null ? 'Not set' : env.alerts_slack_channel,
+          product_id: item.attributes.product.data.attributes.p_id,
```

Comment:
Personally I would pull this out so `const productData = item.attributes?.product?.data?.attributes` then you can refer to these as `product.name`, `product.slug` etc

Avoids a lot of duplication.

The `?` in the object reference will ensure `undefined` is returned rather than the code blowing up because that part of the object doesn't exist.

Reply:
sorted